addons/stock/models/stock_orderpoint.py

```diff
@@ -223,7 +223,7 @@ def action_replenish_auto(self):
         self.trigger = 'auto'
         return self.action_replenish()
 
-    @api.depends('product_id', 'location_id', 'product_id.stock_move_ids', 'product_id.stock_move_ids.state')
+    @api.depends('product_id', 'location_id', 'product_id.stock_move_ids', 'product_id.stock_move_ids.state', 'product_id.stock_move_ids.product_uom_qty')
```

Comment:
Already fix in https://github.com/odoo/odoo/pull/56722

Reply:
I am going to remove the the commit.